client/iocBoot/iocdemo/st_base.cmd

```diff
@@ -0,0 +1,35 @@
+#!../../bin/linux-x86_64-debug/demo
+
+## You may have to change demo to something else
+## everywhere it appears in this file
+
+< envPaths
+
+## Register all support components
+dbLoadDatabase("../../dbd/demo.dbd",0,0)
+demo_registerRecordDeviceDriver(pdbbase)
+
+var(reccastTimeout, 5.0)
+var(reccastMaxHoldoff, 5.0)
+
+epicsEnvSet("IOCNAME", "$(IOCSH_NAME)")
+epicsEnvSet("ENGINEER", "myself")
+epicsEnvSet("LOCATION", "myplace")
+
+epicsEnvSet("CONTACT", "mycontact")
+epicsEnvSet("BUILDING", "mybuilding")
+epicsEnvSet("SECTOR", "mysector")
+
+addReccasterEnvVars("CONTACT", "SECTOR")
+addReccasterEnvVars("BUILDING")
+
+addReccasterExcludePattern("*_", "*__")
+addReccasterExcludePattern("*exclude_this")
+
+## Load record instances
+dbLoadRecords("../../db/reccaster.db", "P=$(IOCSH_NAME):")
+dbLoadRecords("../../db/somerecords.db","P=$(IOCSH_NAME):")
+dbLoadRecords("../../db/test_base.db", "P=$(IOCSH_NAME):")
```

Comment:
Hmm, so this seems to be a copy of `st.cmd`, while the associated removal of `st_test.cmd` to me - without additional context - suggested that this was a rename of that file